pghoard/rohmu/__init__.py

```diff
@@ -30,11 +31,10 @@ def get_class_for_transfer(storage_type):
     raise InvalidConfigurationError("unsupported storage type {0!r}".format(storage_type))
 
 
-def get_transfer(storage_config, *, storage_type=None):
-    # TODO: drop storage_type from the function signature, always read it from the config
+def get_transfer(storage_config):
+    storage_class = get_class_for_transfer(storage_config)
```

Comment:
This will raise a KeyError if `storage_type` isn't defined in `storage_config` so drop the check below.